code-review/README.md

```diff
@@ -8,8 +8,9 @@ Everyone
 
 * Accept that many programming decisions are opinions. Discuss tradeoffs, which
   you prefer, and reach a resolution quickly.
-* Ask questions; don't make demands. ("What do you think about naming this
+* Ask good questions; don't make demands. ("What do you think about naming this
   `:user_id`?")
+* Good questions avoid judgment and avoid assumptions about the author's perspective
```

Comment:
@gylaz good call, added.

@mike-burns not sure, I think the previous line has an example of a good question so it doesn't feel as important to me to have one on this line.